src/types/StopLoss.sol

```diff
@@ -44,33 +44,31 @@ contract StopLoss is BaseConditionalOrder {
         uint32 validityBucketSeconds;
     }
 
-    function getTradeableOrder(
-        address,
-        address,
-        bytes32,
-        bytes calldata staticInput,
-        bytes calldata
-    ) public view override returns (GPv2Order.Data memory order) {
+    function getTradeableOrder(address, address, bytes32, bytes calldata staticInput, bytes calldata)
+        public
+        view
+        override
+        returns (GPv2Order.Data memory order)
+    {
         Data memory data = abi.decode(staticInput, (Data));
-        (, int256 latestSellPrice, , , ) = data.sellTokenPriceOracle.latestRoundData();
-        (, int256 latestBuyPrice, , , ) = data.buyTokenPriceOracle.latestRoundData();
+        (, int256 latestSellPrice,,,) = data.sellTokenPriceOracle.latestRoundData();
```

Comment:
Make sense to do this in a separate PR as it is more involved, but I assume they suggest to use check that the oracle's decimals view method returns the same number for both and check that updated at is not too far in the past (not sure if we should allow the user to specify this though)

Reply:
Yes, I'll put this into a separate PR. tl;dr I'm working on actually generalising the strike price such that it's sell token / buy token in ERC20 decimals, and normalising the returned quote from the oracles.